### ISSUE TITLE
Progress.mip_progress flunks if both LB,UB are negative

### ISSUE THREAD

**Issue by snelson3** [ticdat/ticdat#3]
_From @pjcpjc on February 23, 2019 23:11_

```
verify(lower_bound * 0.99999 <= upper_bound, "lower_bound can't be bigger than upper_bound")
```
a dumbed down version of checking that `lower_bound` is not bigger than `upper_bound`.

Should be a `nearly_same` based check.

_Copied from original issue: opalytics/opalytics-ticdat#216_

**Comment by pjcpjc** [comment]
This was the old issue #196.

**Comment by pjcpjc** [comment]
Fixed at c7ef43edc2a2cc7980b1b2e574ccd7354c4dd96c
